app/templates/frameworks/legal_authority.html

```diff
@@ -0,0 +1,38 @@
+{% extends "_base_page.html" %}
+
+{% block pageTitle %}
+    {{ framework.name }} Legal authority – Digital Marketplace
+{% endblock %}
+
+{% block breadcrumb %}
+    {{ govukBreadcrumbs({
+    "items": [
+      {
+        "text": "Digital Marketplace",
+        "href": "/"
+      },
+      {
+        "text": "Your account",
+        "href": url_for('.dashboard')
+      },
+      {
+        "text": framework.name,
+        "href": url_for(".framework_dashboard", framework_slug=framework.slug)
+      },
+      {
+        "text": "Legal authority to sign"
+      }
+    ]
+  }) }}
+{% endblock %}
+
+{% block mainContent %}
+    {% include 'toolkit/forms/validation.html' %}
+    <form method="POST" novalidate action="{{ url_for(".legal_authority", framework_slug=framework_slug) }}">
+        <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
+        {{ form.legal_authority }}
```

Comment:
Think this needs an `h1`.

Some options off the top of my head:

* Use `govuk_options` from dmutils: https://github.com/alphagov/digitalmarketplace-utils/blob/79cbc5eac674cb438fc90c598b76395f50491a25/dmutils/forms/helpers.py#L70

This would possibly involve bumping dmutils and refactoring the view code, but we might then be able to separate the information.

* Figure out a way to exclude the label from `LegalAuthorityForm`

Can we create a `isPageHeading` flag which for the moment just ignores the question label?

* Repeat the question
Not ideal, but include the question as an `h1` tag as well as at the top of the form.

Reply:
@katstevens @domoscargin OK, I've refactored this in df2278b to use `govuk_options` and `govukRadios`, 
this also adds an `h1` by virtue of the `isPageHeading` option.